packages/smart-contracts/scripts-create2/utils.ts

```diff
@@ -0,0 +1,57 @@
+import { Contract } from 'ethers';
+import * as artefacts from '../src/lib';
```

Comment:
```suggestion
import * as artifacts from '../src/lib';
```
+ all references below